frontend/public/components/instantiate-template.tsx

```diff
@@ -6,6 +6,8 @@ import * as classNames from 'classnames';
 import { ActionGroup, Button } from '@patternfly/react-core';
 
 import { ANNOTATIONS } from '@console/shared';
+
```

Comment:
nit: can we remove empty line

Reply:
@invincibleJai Dropped the new line and rebased it on the master

Reply:
@invincibleJai Please add your nit as a sub-task here: [ODC-3652](https://issues.redhat.com/browse/ODC-3652) we should document anytime we have to say `nit` so we can automate them away.